packages/marko-web-native-x/utils/gtm-events.js

```diff
@@ -0,0 +1,19 @@
+export const pageLoad = (window) => {
+  const dataLayer = window.dataLayerNativeX || [];
```

Comment:
All of these functions should detect the presence of the data layer and do nothing if it doesn't exist